app/app.go

```diff
@@ -272,10 +237,19 @@ func New(
 	app.AuthzKeeper = authzkeeper.NewKeeper(
 		keys[authz.ModuleName], appCodec, app.MsgServiceRouter(),
 	)
+	// Remove the fee-pool from the group of blocked recipient addresses in bank
+	// this is required for the consumer chain to be able to send tokens to
+	// the provider chain
+	bankBlockedAddrs := app.ModuleAccountAddrs()
+	delete(bankBlockedAddrs, authtypes.NewModuleAddress(
```

Comment:
The address of `ConsumerToSendToProviderName` is unnecessary to us, since we're not sending the fee to the provider chain.

Reply:
got it, it does not apply for appchain and should be deleted.